docs/content/docs/reference/retries.md

```diff
@@ -13,7 +13,9 @@ toc = true
 top = false
 +++
 
-Any verb called asynchronously (specifically, PubSub subscribers and FSM states), may optionally specify a basic exponential backoff retry policy via a Go comment directive. The directive has the following syntax:
+Some FTL features allow specifying a retry policy via a Go comment directive. Retries back off exponentially until the maximum is reached.
+
+The directive has the following syntax:
 
 ```go
 //ftl:retry [<attempts=10>] <min-backoff> [<max-backoff=1hr>] [catch <catchVerb>]
```

Comment:
Are these defaults still correct?

Reply:
Yep, I updated the defaults + docs last week